i18n(cn, zh_CN): Add Chinese (Simplified) translation

hedw1gP opened the pull request.

jacobgkau reviewed (commented):
Thank you for the translation. From what I can see, the language code for Chinese is `zh`, so we shouldn't need the identical `cn` files. This can be seen using the output of `localectl list-locales` and confirmed with the [ISO 639-1 standard's list](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). Can you please remove the `cn` files?

hedw1gP commented:
Just removed i18n/cn folder. Thanks for letting me acknowledge this.

By the way, I wonder if `zh_HK` (Chinese Traditional Hong Kong) and `zh_TW` (Chinese Traditional Taiwan) are legit folder name, I switched my Pop!_OS language and `locale` seems to have those values. if that is the case I may do those too.

jacobgkau reviewed (approved):
While testing, this wasn't working because the directory was named `zn_CH` with an underscore. This is correct for how the `LANG` environment variable is set, but it's not valid Project Fluent syntax-- it needs to be `zn-CH` with a hyphen for the directory name. I went ahead and made that change myself, and this is now working.

> By the way, I wonder if `zh_HK` (Chinese Traditional Hong Kong) and `zh_TW` (Chinese Traditional Taiwan) are legit folder name, I switched my Pop!_OS language and `locale` seems to have those values. if that is the case I may do those too.

Those are valid locales. The folder names would be `zh-HK` and `zh-TW` for the same reason as above. In the meantime, user accounts configured for those locales will fall back to the `zn-CH` translation because it's a closer match than English.
